drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -56,7 +56,15 @@ def steps # Don't resolve in batch
         next nil if node.conditional_inflows? # task is conditional, don't show it yet
 
         # initialize step to display in the UI
-        instance.steps.new(node: node).freeze
+        step = instance.steps.new(node: node).freeze
+
+        # If the step is unpersisted, assignees won't be persisted yet either,
```

Comment:
Thanks @gigxz !

I'm curious if you had any thoughts about this (comment from the PR description),
> We can discuss this one more; I think maybe the need to do something like this points to a code smell related to something about the way we are persisting and returning unpersisted steps.

Is this something you want to discuss more, or should we just move forward with what I have here, which works for now?

Reply:
I think moving forward with this is good!